lib/v1/index.js

```diff
@@ -1,14 +1,18 @@
 const { Datatypes, errors, extractDomain } = require('./utils');
 const Config = require('./config');
 const { Crypto, Http, Forms } = require('./core');
-const extractDomainFromUrl = require('./utils/domain');
 
 class EvervaultClient {
-  constructor(teamId) {
+  constructor(teamId, config) {
     if (!Datatypes.isString(teamId)) {
       throw new errors.InitializationError('teamId must be a string');
     }
-    this.config = Config(teamId);
+    if(!config.urls.keysUrl) {
+      if (!Datatypes.isString(config.urls.keysUrl)) {
+        config = undefined;
```

Comment:
If config is not defined correctly, shouldn't we fall back on default values? You're just setting it as undefined here

```suggestion
        config = DEFAULT_CONFIG;
```